src/coupling/generic_assembly.hh

```diff
@@ -59,6 +59,9 @@ template < template<IntDim...> class DimAssembly>
 class GenericAssembly
 {
 private:
+    /// Holds maximum number of neighbouring element adding in one step to patch (own element and its side, coupling, boundary neighbours).
+    static const unsigned int maximal_neighbouring_elem = 9;
```

Comment:
We can remove this arbitrary limit, but we have to check that number of ghost elements of the patch fits into ElementCacheMap.